resharper/resharper-unity/src/Rider/CodeInsights/RiderUnityImplicitUsageHighlightingContributor.cs

```diff
@@ -87,12 +87,12 @@ private void AddHighlighting(IHighlightingConsumer consumer, AbstractUnityCodeIn
             var extraActions = new List<CodeLensEntryExtraActionModel>();
             if (!myConnectionTracker.IsConnectionEstablished())
             {
-                extraActions.Add(new CodeLensEntryExtraActionModel("Unity is off", null));
+                extraActions.Add(new CodeLensEntryExtraActionModel("Unity is not running", null));
```

Comment:
No! 
Use shorter string, because there is no space for long strings in context action for code vision